app-web/src/utils/hooks.js

```diff
@@ -51,7 +51,15 @@ export const useSearch = query => {
   const [results, setResults] = useState([]);
   const index = searchClient.initIndex(`Devhub-Algolia-${ALGOLIA_INDEX_SUFFIX}`);
   useEffect(() => {
-    if (query) {
+    if (query && query.includes('persona:')) {
+      index
```

Comment:
the search operation is the same. I think the code would look cleaner if we just did the search once and modified the arguments inside of it. 

```js

if foo
query = '';
options = {
  facetFilters: bar
}
else 
query = blah
options = {...}

// do search here